packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -642,6 +644,28 @@ const BarfTurnTasks: GarboTask[] = [
       sobriety: "sober",
     },
   ),
+  wanderTask(
+    "freerun",
+    {
+      equip: $items`spring shoes`,
+      bonuses: new Map([[$item`carnivorous potted plant`, 500]]),
+    },
+    {
+      name: "Spring Shoes Freerun",
+      ready: () =>
+        have($item`spring shoes`) &&
+        romanticMonsterImpossible() &&
+        (getWorkshed() !== $item`model train set` ||
+          TrainSet.next() !== TrainSet.Station.GAIN_MEAT),
```

Comment:
Meat is special cased because it disappears when we run, not because all train stuff disappears when we run and meat is the best, right?